package.json

```diff
@@ -10,16 +10,12 @@
     "lint:css": "stylelint src",
     "test": "vue-cli-service test:unit --maxWorkers=1",
     "test-ci": "vue-cli-service test:unit --reporters=default --reporters=jest-junit --maxWorkers=1",
-    "docs": "vue-styleguidist server",
-    "docs:build": "vue-styleguidist build"
+    "docs": "vue-cli-service styleguidist server",
+    "docs:build": "vue-cli-service styleguidist build"
```

Comment:
Sorry about the delay. The project now compiles - but running `npm run docs:build` now seems to run the server.

Reply:
You are right; I did not read my own docs... hum

`styleguidist` does not take any extra arguments.

I should have used 

- `syleguidist server` -> `styleguidist`
- `styleguidist build` -> `styleguidist:build`

I pushed a fix to my branch.

Sry